### PR TITLE
update online maps in map view

### PR DESCRIPTION
**Pull request by hakuchi**
Fixed tile server to https and/or to new links.     
Deleted Thunderforest maps because of missing api key.     
Deleted Openmapsurfer and Mapquest because I was not able to find a free tile server.
Also fixing some translations

### CODE REVIEW

**Comment by bekuno** [comment]
OK to test

File: src/main/java/menion/android/whereyougo/maps/mapsforge/mapgenerator/MapGeneratorFactory.java [hunk 1]
```diff
@@ -83,7 +83,7 @@ public static MapGenerator createMapGenerator(MapGeneratorInternal mapGeneratorI
             case ESRI_WORLD_IMAGERY:
                 return new FormatURLTileDownloader(
                         18,
-                        "https://services.arcgisonline.com/arcgis/rest/services/World_Imagery/MapServer/tile/{z}/{y}/{x}.png",
+                        "https://services.arcgisonline.com/ArcGIS/rest/services/World_Imagery/MapServer/tile/{z}/{y}/{x}.png",
```
Comment (bekuno) [inline src/main/java/menion/android/whereyougo/maps/mapsforge/mapgenerator/MapGeneratorFactory.java]: The original address is fine. This is the form wich are used on the server, see at https://services.arcgisonline.com/arcgis/rest/services/World_Imagery/MapServer
The paths are context insensitive.
Please revert.

File: src/main/res/values-de/strings_pref.xml [hunk 2]
```diff
@@ -42,6 +42,10 @@
     <string name="pref_guiding_waypoint_sound_beep_on_distance">Signalton nach Entfernung</string>
     <string name="pref_guiding_waypoint_sound_custom_on_distance">Nutzerdefinierter Signalton nach Entfernung</string>
     <string name="pref_guiding_waypoint_sound_increasing">Frequenz erhöhen, wenn man näher kommt</string>
+    <string name="pref_guiding_zone_point">Zonen-Navigationspunkt</string>
+    <string name="pref_guiding_zone_point_center">Vom Autor definiere Zonenmitte</string>
```
Comment (bekuno) [inline src/main/res/values-de/strings_pref.xml]: definierte

File: src/main/res/values-de/strings_pref.xml [hunk 3]
```diff
@@ -64,6 +68,8 @@
     <string name="pref_root_desc">Wähle ein Zielverzeichnis für Wherigo.</string>
     <string name="pref_save_game_auto">Spiel automatisch speichern</string>
     <string name="pref_save_game_auto_desc">Das Spiel wird automatisch gespeichert werden, wenn du die Anwendung minimierst.</string>
+    <string name="pref_save_game_slots">Spielestände</string>
```
Comment (bekuno) [inline src/main/res/values-de/strings_pref.xml]: Spielstände

**Comment by bekuno** [comment]
Do we need further api keys in addition to thunderforest?
Please let them integrated.
We should do the api usage as in c:geo. (user with his developer key, in CI integration of the project key)

**Comment by Lineflyer** [comment]
The string updates seem unrelated to the PR topic. Why not seperate into another PR or enforce the Crowdin solution instead?

**Comment by kumy** [comment]
retest this please

**Comment by hakuchi** [comment]
@bekuno In c:geo are Google-Street, Google-Satellite OSM and offline map for using as background.
Why should WherYouGo have more choises?

Most of these backgrounds use data from OSM, so the basemap is always the same and give the same good or bad help during playing a wherigo.

After this changes the selection looks manageable and you can easily switch between the backgrounds.

@Lineflyer I know that the translations are unrelated to the PR. Sry for that, but I changed all of this in my fork to make my own apk. 
On Crowdin is nothing to translate. Only emty languages.

**Comment by bekuno** [comment]
@hakuchi Many thanks for your work. Every development is very welcome.

First of all - this is a completely voluntary project. All work is done in the spare time of each of us. And we all have a "real life" outside this project(s).
The first goal for WhereYouGo is to get the app running again. After that all further things will be done. 
The crowdin configuration has to wait until someone (me?) has time to configure it properly.

For your work - it would be fine if you use different branches for different things. Normally, the master branch of your fork ("origin") should be synchronized with the master branch of the "upstream" branch. (cgeo\whereyougo\master). (pull from upstream and push to the origin) 
You can use a separate branch to develop your complete app and create your own apk. 
To create a PR, you should first create a branch from the current master, edit there and then push the changes to origin. This way you can create a PR for this change only. 
For (non-trivial) changes - please create an issue first to describe the problem. So it is possible to discuss this independently of a PR.

**Comment by mucek4** [comment]
retest this please

**Comment by bekuno** [comment]
@hakuchi 
I tested your changes for the map providers.
This looks very good to me.

To bring it into WhereYouGo - please create a new branch (eg. mapp_change) from your master. Then revert all changes that are not part of the map adjustments, squash the commits. 
As second create a branch from your master eg. as "my_development_tree" for your personal work.
After that close this PR. Then you should revert your changes on your master, then pull master from upstream. Push master to origin (force). (delete and pull should also work, I think)
Finally rebase the "map_change" branch to master. Now you can create the PR once more.

As long as crowdin is not filled from the code base you can add all changed language files within the PR. After crowdin is established then only english changes are relevant, other will be overwritten after crowdin sync.

**Comment by hakuchi** [comment]
> @hakuchi
> I tested your changes for the map providers.
> This looks very good to me.

Ok thenn merge it to cgeo/master

> 
> To bring it into WhereYouGo - please create a new branch (eg. mapp_change) from your master. Then revert all changes that are not part of the map adjustments, squash the commits.
> As second create a branch from your master eg. as "my_development_tree" for your personal work.
> After that close this PR. Then you should revert your changes on your master, then pull master from upstream. Push master to origin (force). (delete and pull should also work, I think)
> Finally rebase the "map_change" branch to master. Now you can create the PR once more.

If this changes are OK, they can merged into the master and if not this should be closed without merging.      
In future PR I can doo this, but not for this.

It makes no sense to spend more time into changing and repushing this to different PR to make it possible to get merged into the cgeo/master.

If somone from the cgeo team will spend his time into splitting this into different PR, feal free to doo this.

I deffinitly willl not spend more time into this changes, when there is noting wrong in this code!

**Comment by Lineflyer** [comment]
At this point of time adding strings in several languages should still be fine as we did not fill Crowdin yet.

Squashing can be done while merging here on the Github UI.
The only thing we really lose is the commit information. I could live with that, if all changes of this PR are considered OK by others.

**Comment by Lineflyer** [comment]
@bekuno Feel free to merge if you are satisfied with the change as such (ignoring for now, that topics are mixed). IMHO we should include the map provider update before releasing to production

**Comment by Lineflyer** [comment]
I fetched, rebased and squashed the commits from hakuchi/master and will provide a new PR.
Thanks @hakuchi for your work.

**Pull request closed without merging**